lib/katello/plugin.rb

```diff
@@ -218,8 +218,9 @@
        :parent => :lab_features_menu,
        :turbolinks => false
 
-  allowed_template_helpers :subscription_manager_configuration_url, :repository_url
+  allowed_template_helpers :subscription_manager_configuration_url, :repository_url, :sortable_version
   extend_template_helpers Katello::KatelloUrlsHelper
+  extend_template_helpers Katello::KatelloVersionHelper
   extend_template_helpers Katello::Concerns::BaseTemplateScopeExtensions
```

Comment:
Should we just add this method to the base template scope extensions? considering it is not actually a helper that is used elsewhere.

Reply:
Do you mean to move this into core? There was actually NACK on this thus I am sending my PR into Katello.

Reply:
No, i mean instead of adding another module, add the new method in Katello::Concerns::BaseTemplateScopeExtensions which is where all the template extensions live